tools/pot/openvino/tools/pot/graph/passes.py

```diff
@@ -684,7 +690,7 @@ def create_bias_node(graph: Graph, src_node):
     add_bias_shape = [1] * len(bias_shape)
     add_bias_shape[1] = bias_shape[1]
     add_bias = Const(graph,
-                     {'value': np.zeros(add_bias_shape, dtype=np.float32),
+                     {'value': np.zeros(add_bias_shape, dtype=src_node.out_port(0).get_data_type()),
```

Comment:
@AlexKoff88, can we assume here that the Bias precision would be the same as the output precision of Convolution/MatMul nodes? Or the Bias precision could be the same as weights?

Reply:
In most cases, it has the precision of an accumulator. I would recommend keeping it FP32 even so it is likely FP16 in the FP16 compressed models. It is up to us actually.